1000genomes_phase3/README.md

```diff
@@ -0,0 +1,161 @@
+<!-- R Markdown Documentation, DO NOT EDIT THE PLAIN MARKDOWN VERSION OF THIS FILE -->
+
+<!-- Copyright 2016 Google Inc. All rights reserved. -->
+
+<!-- Licensed under the Apache License, Version 2.0 (the "License"); -->
+<!-- you may not use this file except in compliance with the License. -->
+<!-- You may obtain a copy of the License at -->
+
+<!--     http://www.apache.org/licenses/LICENSE-2.0 -->
+
+<!-- Unless required by applicable law or agreed to in writing, software -->
+<!-- distributed under the License is distributed on an "AS IS" BASIS, -->
+<!-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. -->
+<!-- See the License for the specific language governing permissions and -->
+<!-- limitations under the License. -->
+
+1,000 Genomes Phase 3 Variants
+==============================
+
+### Additional Resources
+* [Schema](https://bigquery.cloud.google.com/table/genomics-public-data:1000_genomes_phase_3.variants_20150220_release?pli=1)
+* [Data Provenance](http://googlegenomics.readthedocs.io/en/latest/use_cases/discover_public_data/1000_genomes.html)
+* [Index of variant analyses](./sql)
+
+### Metrics
+
+In the following plots we display metrics computed on both 1,000 Genomes phase 1 and phase 3 variants.
+
```

Comment:
knitr put those in.  This file is auto-generated.

In the markdown in this repo, I suppress most/all of the echoing of R code.

In the codelabs repo, I echo nearly all the R code.